src/VCR/LibraryHooks/SoapHook.php

```diff
@@ -69,8 +69,18 @@ public function doRequest($request, $location, $action, $version, $one_way = 0,
         }
 
         $vcrRequest = new Request('POST', $location);
-        $contentType = ($version == SOAP_1_2) ? 'application/soap+xml' : 'text/xml';
-        $vcrRequest->setHeader('Content-Type', $contentType . '; charset=utf-8; action="' . $action . '"');
+
+        switch ($version) {
+            case SOAP_1_1:
+                $vcrRequest->setHeader('Content-Type', 'text/xml; charset=utf-8;');
+                $vcrRequest->setHeader('SOAPAction', $action);
+                break;
+
+            case SOAP_1_2:
+                $vcrRequest->setHeader('Content-Type', sprintf('application/soap+xml; charset=utf-8; action="%s"', $action));
+                break;
```

Comment:
@laland I like how you moved out the ternary conditional!

What happens when there is another SOAP version coming out? I'd either throw an exception or use the SOAP 1.2 behaviour for other versions (preferred).
Maybe just use a 
`if (SOAP_1_1) {...} else { // >=SOAP_1_2 ...}`